bin/backend_check

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```

Comment:
Does this file need to be made executable?

Reply:
Already is